src/main/kotlin/lotto/domain/LottoGameMoney.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.domain
+
+const val PRICE_OF_LOTTO = 1000
+private val GAME_MONEY_REGULAR_EXPRESSION = "(\\d*)".toRegex()
+
+data class LottoGameMoney(val lottoGameMoney: Int) {
+
+    fun getCountOfGame(): Int = lottoGameMoney / PRICE_OF_LOTTO
+
+    companion object {
+        fun from(lottoGameMoney: String): LottoGameMoney? {
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-lotto/pull/26#discussion_r468326731

```suggestion
        fun from(lottoGameMoney: String): LottoGameMoney {
```

Reply:
try-catch 를 이용하지 않고 null 로 처리해서, 예외처리 하는 부분을 반영한 사항인데, null 로 반환되는 것이 더 큰 리스크라 코멘트 주신건지 궁금합니다.

참고로 코틀린에서는 아래와 같이 예외 처리를 한다. 장기적으로는 아래와 같이 예외 처리하는 걸 연습해 본다.
논리적인 오류일 때만 예외를 던진다.
논리적인 오류가 아니면 예외를 던지지 말고 null을 반환한다.
실패하는 경우가 복잡해서 null로 처리할 수 없으면 sealed class를 반환한다.
일반적인 코틀린 코드에서 try-catch를 사용하지 않는다.